src/widgets/modals/CashTransactionModal.js

```diff
@@ -48,12 +48,10 @@ export const CashTransactionModal = ({ onConfirm }) => {
     [isCashIn]
   );
 
-  const isValidTransaction = useMemo(() => !!name && !!amount && (isCashIn || !!reason), [
-    name,
-    amount,
-    reason,
-    isCashIn,
-  ]);
+  const isValidTransaction = useMemo(
+    () => !!name && !!amount && amount > 0 && (isCashIn || !!reason),
```

Comment:
if amount was `0` then `!!amount == false` and this expression is the same as before, no?

Reply:
It explicitly coerces the `amount` to an `int`. Not a nice solution, but under time constraints does the job.

Reply:
It explicitly coerces the `amount` to an `int`. Not a nice solution, but under time constraints does the job.

Reply:
24ccda1 (temporary fix, needs a refactor asap).